src/models/Settings.js

```diff
@@ -530,4 +533,21 @@ export default Model.extend({
   isDsTheme: function() {
     return false;
   },
+
+  unsetOieSettings: function() {
```

Comment:
When the transaction is canceled, a new one should be started when back to the login screen. No preserve logic is needed here.

Reply:
After starting login flow these 3 options are unset:
https://github.com/okta/okta-signin-widget/blob/8d27ecb7f46fb1ea11ad49cf8538b661de546239/src/v2/BaseLoginRouter.js#L216-L218
This will prevent further interaction:
https://github.com/okta/okta-signin-widget/blob/8d27ecb7f46fb1ea11ad49cf8538b661de546239/src/v2/client/startLoginFlow.js#L37-L40

Reply:
`stateToken` should be unset in a new transaction. `proxyIdxResponse` is probably used for debugging only. I only setting you want to preserve is `useInteractionCodeFlow`, which can be set to true programmatically when an event is triggered. We can probably name it as `interactionCanceled`.